arcade/gl/vertex_array.py

```diff
@@ -240,7 +240,7 @@ def _build(
 
     def render(
         self, mode: GLenumLike, first: int = 0, vertices: int = 0, instances: int = 1
-    ):
```

Comment:
```suggestion
    ) -> None:
```
I think you may have meant this.